pixelot/src/app/left-sidebar/left-sidebar.component.ts

```diff
@@ -227,17 +227,39 @@ export class LeftSidebarComponent {
 
   }
 
-  openLayerDialog(): void {
+  openLayerDialog(type: string): void {
     const dialogRef = this.dialog.open(AddLayerDialog, {
       width: '500px',
     });
 
+    if (type == 'tilemap') {
+      dialogRef.componentInstance.path_required = true;
+    }
+
     dialogRef.afterClosed().subscribe(result => {
       if (!result)
         return;
+      
+      console.log(result);
+
+      let new_layer: engine.RenderLayer;
+
+      if (type == 'tilemap') {
+        if (result.path == '') {
+          this._snackBar.open('Please enter a path', 'Close', {
+            duration: 5000
+          });
+          return;
```

Comment:
I get console errors if I don't include a path or include an invalid json file. It looks like you've added the error handling in place here but for some reason it's not triggering?